### PR TITLE
Add 172b-exp2 card

### PR DESCRIPTION
**Pull request by odashi**
Settings is fixed according to the discussion on the model WG.

Important changes from v3:

* Use v3.1 corpus: fix whitespace issues on WARP
* Change initial learning rate from 1e-4 to 8e-5
* **Change Adam epsilon from 1e-5 to 1e-8**

Discussion:
https://docs.google.com/document/d/1AY_yhyBCWqpQTHvXKjk-m5BTXByedrDZzDOgjDw6GoQ/edit#heading=h.39oxz76vdp54

This PR also adds a row for final LR for each card.

### CODE REVIEW

File: models/geniac-172b-exp2.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+|Vocabulary size|99,487|
+|Tokenizer|[llm-jp-tokenizer-100k.ver3.0b1.model](https://github.com/llm-jp/llm-jp-tokenizer/blob/870a27ce6872e105e4b76cdf2e68c8b7ebfc6a37/models/ver3.0/llm-jp-tokenizer-100k.ver3.0b1.model)|
+|max_position_embeddings|4096|
+|Positional embedding|RoPE|
+
+# Dataset specs
+|Spec name|Value|
+|:---|:---|
+|Dataset|LLM-jp v3.1|
+|Number of tokens|**FIXME**|
```
Comment (jnszk) [inline models/geniac-172b-exp2.md:23]: 最終的なデータ量が確定したら ** FIXME **を修正する

geniac-13B-exp4 と同じ値になるはず

File: models/geniac-172b-exp2.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+|Final learning rate|8e-6|
+|Beta1|0.9|
+|Beta2|0.95|
+|Epsilon|1e-8|
+|Weight decay factor|0.1|
+|Warmup strategy|Linear|
+|Warmup steps|2000|
+|Learning rate scheduling strategy|Cosine|
+|Learning rate Decay steps|**FIXME**|
+|Total training steps|**FIXME**|
```
Comment (jnszk) [inline models/geniac-172b-exp2.md:41]: 最終的なデータ量が確定したら ** FIXME **を修正する

Learning rate Decay stepsとTotal training stepsは同じ値とする

**Review by jnszk — CHANGES_REQUESTED** [review]

**Review by jnszk — DISMISSED** [review]

File: models/geniac-172b-exp2.md (hunk 2 above)
Comment (odashi) [reply models/geniac-172b-exp2.md]: memo: total = warmup + decay なので2,000だけ差があります

**Comment by odashi** [comment]
@jnszk もう一度approveお願いします

**Review by jnszk — APPROVED** [review]
